hadoop-common-project/hadoop-nfs/src/main/java/org/apache/hadoop/portmap/RpcProgramPortmap.java

```diff
@@ -60,6 +60,8 @@
   private final ChannelGroup allChannels;
 
   RpcProgramPortmap(ChannelGroup allChannels) {
+    super(1, 1, 1);
+    // FIXME: set default idle timeout 1 second.
```

Comment:
Why setting timeout to 1 second?  Is this for debugging?

Reply:
So this one I am not 100% sure. Without calling super(), the default is no idle timeout (not sure about the default behavior in netty3), TestPortmap#testIdle() will fail. So I have to set a timeout. If I set a long timeout like 60 seconds,  TestPortmap#testIdle() will fail too.